tests/extended/test_trainer_ext.py

```diff
@@ -366,12 +366,11 @@ def run_trainer(
                 n_gpus_to_use = get_gpu_count()
             master_port = get_torch_dist_unique_port()
             distributed_args = f"""
-                -m torch.distributed.launch
                 --nproc_per_node={n_gpus_to_use}
                 --master_port={master_port}
                 {self.examples_dir_str}/pytorch/translation/run_translation.py
             """.split()
-            cmd = [sys.executable] + distributed_args + args
+            cmd = ["torchrun"] + distributed_args + args
```

Comment:
If this one is changed, the test will need to be decorated to require a more recent version of PyTorch.